src/components/task/create-new/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from "react";
+import { useForm } from "react-hook-form";
```

Comment:
couldn't we use react final form instead of react hook form?

Reply:
Yeah vai we can. While choosing form-validation library I found `react-hook-form` has higher acceptance, less issues & provides regular updates than the similar libraries. Though it won't put much effect into this project but it was my motivation behind selecting this library. @ray-wtag vai.
![Screenshot 2023-04-12 at 2 42 57 PM](https://user-images.githubusercontent.com/128278816/231404742-c2807c35-7a36-49f6-8108-6a662243c364.png)